lib/views/login_register/login.dart

```diff
@@ -15,11 +52,7 @@ class LoginScreen extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.start,
         children: <Widget>[
           const Padding(
-            padding: EdgeInsets.only(
-              top: 50,
-              bottom: 20,
-              left: 20,
-            ),
+            padding: EdgeInsets.fromLTRB(20, 50, 0, 20),
```

Comment:
I like using the `.fromLTRB()` constructor when most of the padding values are being set.